vignette/montlake_seattle.Rmd

```diff
@@ -0,0 +1,240 @@
+---
+title: "Reproducing Montlake Eastside Seattle, US"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Reproducing Montlake Eastside Seattle, US}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+author: 
+- "Robin Lovelace, Trevor Nederlof and Nathanael Sheehan"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+## Introduction 
+Lets start with the city A/B street began, Seattle. U.S. The seaport city is home to over 700,000 people, including A/B Street creator, Dustin Carlino, who has been developing tools to empirically study the impact of small changes within the road network: this means "you can transform that street parking into a bus lane or fix that pesky left turn at a traffic signal, measure the effects, then propose actually making the change". For the past two years, Seattle has been a key are of study within the A/B street simuverse and thus makes it a great starting point in understanding the utilities of `abstr` to generate site data for A/B street. 
+
+<!-- ## Install packages -->
+<!-- ```{r, eval = FALSE} -->
+<!-- devtools::install_cran(tidyverse) -->
+<!-- ``` -->
+
+## Example
+This example demonstrates how to wrangle the data for the three key components needsed to generate scenarios for A/B street. These components are: OD data, site zones and site buildings. With these components, the `abstr` package is ready to convert dataframes into simulations! Lets get started.
+
+### Load packages
+```{r, eval = FALSE}
```

Comment:
Good solution to the pkg dependency conundrum. Tidyverse is too easy to use and popular not to recommend...

Reply:
I believe we had a similar issue with PCT - should we also generate an rpubs with plots?

Reply:
> I believe we had a similar issue with PCT - should we also generate an rpubs with plots?

It's a balance between reproducibility and package dependencies and run times. There are good balances to be found most of the time but when you want to process a load of data using `eval=FALSE` and linking to an externally hosted file that shows the whole thing like RPubs or even a GitHub issue or gist is a good shout. 

Partial solution to the `tidyverse` conundrum, compromise and Suggest `dplyr`:

https://github.com/a-b-street/abstr/blob/a0d2507b1157eeceda1f5f7819703e55867016fd/DESCRIPTION#L41-L42